lib/python/ensembl/brc4/runnable/compare_report.py

```diff
@@ -42,7 +42,7 @@ def run(self):
         report = output_dir + "/report.log"
         print("Write report in %s" % report)
         
-        fields = ("species", "seq_count_1", "seq_count_2", "diff_length", "common", "only1", "only2", "max_only1", "max_only2")
+        fields = ("species", "seq_count_1", "seq_count_2", "diff_length", "common", "only1", "only2", "max_only1", "max_only2", "other_locations", "summary", "organellar_summary")
```

Comment:
```suggestion
        fields = (
            "species", "seq_count_1", "seq_count_2", "diff_length", "common", "only1", "only2", "max_only1",
            "max_only2", "other_locations", "summary", "organellar_summary"
        )
```
Always aim to a soft line length of 100 chars, and a hard limit of 110.